Added option for including symbols (PDBs) from MSVC.

RFerraro opened the pull request:
As mentioned in #9:

I'm building a project that uses the gtest package and I would like to avoid LNK4099 (missing gtest.pdb) warnings when building with MSVC.

This PR has an implementation that's worked for me, and its a fairly minor change. Maybe the option could be named better, but you know: 

> There are only two hard things in Computer Science: cache invalidation and naming things.
> -- Phil Karlton

lasote commented:
Great, thanks. Just an observation. Should be better to remove the option if it's not compiled with Visual Studio. It has no sense an option related with VS if we are, for example, in linux. You can do something like this:

```
def configure(self):
    if self.settings.compiler != "Visual Studio":
        try:
            self.options.remove("include_pdbs")
        except: 
            pass
```

Then in package method:


```

def package(self):
    ...

    if self.settings.compiler == "Visual Studio" and self.options.include_pdbs:
        self.copy(pattern="*.pdb", dst="lib", src=".", keep_path=False)

```

Could you change it and see if CI is ok? Thanks for your collaboration! :+1:

RFerraro commented:
I added the changes from your feedback, and added some code to the package tools builder so the CI will build the added option in its states: (default, true, false).

lasote commented on build.py:
```diff
@@ -4,5 +4,21 @@
+            noPdbOptions = options.copy()
+            
+            pdbOptions.update({"gtest:include_pdbs": "True"})
+            noPdbOptions.update({"gtest:include_pdbs": "False"})
```
I think this build "noPdbOptions" should not be added, the default is already False, so the "options" build is the same build.

RFerraro replied:
Done, see: [122460d](https://github.com/lasote/conan-gtest/pull/10/commits/122460d863d64eb89a96cf5be5890e1dfcee7032). I removed the redundant option and simplified the branch condition in the loop.

lasote commented:
Thank you :+1:  :+1:
